src/ignitus-Shared/ignitus-DesignSystem/ignitus-Assets/ignitus-Icons/pollIcon/poll.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import {HTMLAttributes} from 'react';
+type Props = HTMLAttributes<SVGElement>;
+
+const PollIcon = ({
```

Comment:
wrong icon that's why you have to define different heigh/width here
`export const StyledIcon2 = styled(Icon)`

Use this icon https://material.io/resources/icons/?search=poll&icon=poll&style=round

I have updated your pr feelf ree to check new icons.